src/test/java/net/datafaker/FakerTest.java

```diff
@@ -277,10 +277,10 @@ void differentLocalesTest() {
     @Test
     @Timeout(value = 3, unit = TimeUnit.MINUTES)
     void issue883Test() throws InterruptedException {
-        for (int i = 0; i < 10_000_000; i++) {
+        for (int i = 0; i < 10_000; i++) {
             Faker f = new Faker();
             String s = f.ancient().god();
-            if (i % 1_000_000 == 0) {
+            if (i % 1_000 == 0) {
```

Comment:
This was purposefully huge to ensure a timing/performance issue isn't re-introduced.

Reply:
If it's really going to be dropped by a factor of 10 then the timeout should as well I guess.

Reply:
We can restore `10_000_000`, of course. It just seemed to much for me. Was a bit slow on my machine.

Reply:
I believe that's the right move.

Reply:
Before changing number here we need to have a good replacement for that
Just changing `10_000_000` to `1_000` makes test useless since with so small number it is unable to reproduce the original problem https://github.com/datafaker-net/datafaker/issues/883  for the code before it was fixed